tests/unit_tests/conditional/ci/test_categorical_test.py

```diff
@@ -0,0 +1,156 @@
+import numpy as np
+import pandas as pd
+from numpy.testing import assert_almost_equal
+
+from dodiscover.ci import CategoricalCITest
+
+df_adult = pd.read_csv("dodiscover/testdata/adult.csv")
+
+
+def test_chisquare_adult_dataset():
```

Comment:
Optimally, these would be multiple tests with descriptive test names. For instance, the "given-when-then"-pattern is quite common for that: ```def given_unconditionally_independent_variables_when_perform_categorical_ci_test_with_pearson_then_returns_expected_test_statistic()```. This may look ridiculously long, but in case of unit tests, that is fine and probably less work then adding more descriptions to it.

Here, maybe you can at least make one test for the unconditional and one test for the conditional setting.

Reply:
Done.